astrodynamics-fall-2018/astrodynamicsFall2018PDD.tex

```diff
@@ -0,0 +1,372 @@
+
+    % Note that math terms and non-math terms are separated and alphabetized, regardless of the order in which they are defined. (Recall terms \$like this\$ are in the math environment)
+    % Read more about advanced nomenclature formatting here:\\
+    % \url{https://www.sharelatex.com/learn/Nomenclatures}
+  }
+\nomenclature{RIT}{Rochester Institute of Technology}
+\nomenclature{SPEX}{RIT Space Exploration}
+\nomenclature{PDD}{Project Design Document}
+\nomenclature{CCD{}Charged Coupled Device}
+\nomenclature{MSD}{Multidisciplinary Senior Design}
+\nomenclature{SG}{Student Government}
+\printnomenclature{}
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{R}{IT} SPEX has direct ties to the RIT Observatory though one of the primary advisors and a mutual interest in astronomy.
+Despite this, both groups have had little collaboration.
+The RIT SPEX Astrodynamics team has some equipment that is measurable to the equipment in the observatory, but has trouble with operation and lack of experience.
+The RIT Observatory has a need for students to work on engineering projects, which RIT SPEX Astrodynamics can provide.
+RIT SPEX has always had members interested in observing using various telescopes, but these members have little training.
+There have already been projects that gave the team insight into the equipment behind observing.
+These include: building and modifying a rolling mount for its 12" Meade tele-scope, building mounts for an Orion sighting scope, building a barn door camera mount, and general maintenance on all telescopes.
+The team is looking for more projects similarly to expand their skill sets.
+
+\section{Primary Objective}
+\label{sec:primary-obj}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+
+  There are three projects selected to be worked on in the fall semester.
+  These projects are the first steps towards improving the observatory’s capabilities and a convenient fully functioning automated dome system.
```

Comment:
replace `observatory’s` with `observatory's`. the apostrophe here is a special character (left facing apostrophe) but latex is looking for a single quote `'`

do a find/replace all on this character: `’` with this character: `'`